tests/test_ec2.py

```diff
@@ -1,13 +1,16 @@
 from ocw.lib.EC2 import EC2, Provider
 from webui.settings import PCWConfig
 from tests.generators import mock_get_feature_property
-from tests.generators import min_image_age_hours, max_image_age_hours, ec2_max_volumes_age_days, \
-    ec2_max_snapshot_age_days
+from tests.generators import ec2_max_age_days
+from faker import Faker
```

Comment:
Why use `faker`, i.e. another dependency instead of the build-in [unittest.mock](https://docs.python.org/3/library/unittest.mock.html) ?

Reply:
Probably beyond the scope of this already big PR.